include/autocluster.hrl

```diff
@@ -28,6 +28,7 @@
          {config, cluster_name,          "CLUSTER_NAME",           "undefined",  string,  false}, %% Consul && etcd
 
          {config, consul_acl_token,      "CONSUL_ACL_TOKEN",       "undefined",  string,  false}, %% Consul
+         {config, consul_allow_warn,     "CONSUL_ALLOW_WARN",      false,        atom,    false},
```

Comment:
@michaelklishin agree, in hindsight the current name may also cause confusion with `CLEANUP_WARN_ONLY` option. Happy to go with the suggested name, as can't think of anything shorter to convey the meaning